packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -53,6 +54,32 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
       console.warn(`[${componentName}] \`${main}\` and \`${sub}\` cannot be used together. \`${sub}\` will be ignored.`)
     }
   }
+  const resolveTrailingSlashBehavior = (
+    to: RouteLocationRaw,
+    resolve: (to: RouteLocationRaw) => RouteLocation & { href?: string }
+  ): RouteLocationRaw | RouteLocation => {
+    if (options.trailingSlash !== 'append' && options.trailingSlash !== 'remove') {
+      return to
+    }
+
+    const forceAppend = options.trailingSlash === 'append'
+    if (typeof to === 'string') {
+      console.log('withTrailingSlash', to, withTrailingSlash(to, true))
```

Comment:
```suggestion
```

Reply:
btw, just asking: why did you disable eslint rule `no-console`?

Reply:
we emit console logs deliberately in lots of places